scripts/weather_forecast/readme.md

```diff
@@ -1,5 +1,21 @@
-a script to get weather forecast. 
-requirements:
-1: geopy
-usage :
-python forecast.py city_name
+# weather_forecast
+## About
+A Python script to get weather forecast for a particular city. 
+## Setup
+**Step1:** Install Python3 [Link](https://github.com/geopy/geo)  
+**Step2:** Install geopy library [Link](https://github.com/geopy/geo)
+```
+pip install geopy
+```
+**Step3:** Clone this repository
+```
+https://github.com/GDSC-RCCIIT/General-Purpose-Scripts.git
+```
+**Step4:** Change the directory
+```
+cd General-Purpose-Scripts\scripts\weather_forecast
```

Comment:
I would prefer if you use forward slash instead of backslash on this one.

Reply:
I changed it